scripts/apps/search/styles/search.scss

```diff
@@ -541,4 +541,17 @@
 
         }
     }
+}
+
+
+// used to resize thumbnail of item while dragging it
+#drag-image-holder {
+    position: absolute;
+    left: -2000px;
+    float: left;
```

Comment:
Do you need a float for some reason? It works fine for me without it, at least in chromium.

You should probably limit `max-height` too since if you have a `200 x 800` image it won't look good either.